app/views/vacancies/index.html+phone.haml

```diff
@@ -1,4 +1,12 @@
 - content_for :page_title_prefix, t('jobs.heading')
+= content_for :head_links do
```

Comment:
Could the duplication between the 2 views be fixed with a partial?

Reply:
Good spot, I didn't actually know that this is what partials were for. I've made the change in https://github.com/DFE-Digital/teacher-vacancy-service/pull/983/commits/82ce1e8f17dbcef3e8eb058da54d0d36ccf09b92